prac_07/project.py

```diff
@@ -0,0 +1,17 @@
+"""
+CP1404 Practical 7 - Project class file.
+"""
+
+class Project:
+    """Represent what will be in a Project object."""
+
+    def __init__(self, name, start_date, priority, cost_estimate, completion_percentage):
```

Comment:
Did you consider having a Docstring here?

Reply:
Oh, I totally forgot. I'll fix it now.